.github/actions/schema-validator/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:12.9.0
```

Comment:
Since we are using v14 in the workflow itself, maybe we can up the version here as well.

Reply:
Node version has been updated to v14.